languages/julia/config.json

```diff
@@ -8,6 +8,47 @@
         "indent_style": "space",
         "indent_size": 4
     },
+    "tags": [
+        "compiles_to/binary",
+        "typing/dynamic",
+        "paradigm/declarative",
+        "paradigm/functional",
+        "paradigm/imperative",
+        "paradigm/object_oriented",
+        "paradigm/procedural"
+    ],
+    "key_features": [
+        {
+            "title": "Fast",
+            "content": "Julia is designed for high performance. Code compiles to efficient native code for various platforms",
+            "icon": ""
+        },
+        {
+            "title": "Interactive",
+            "content": "Julia is dynamically typed, feels like a scripting language and has good support for interactive use",
+            "icon": ""
+        },
+        {
+            "title": "Scientific",
+            "content": "Julia is designed with reproducibility, distributed computing and easy collaboration in mind",
+            "icon": ""
+        },
+        {
+            "title": "Composable",
+            "content": "Julia uses multiple dispatch as a paradigm, making it unreasonably effective at composing programs",
+            "icon": ""
+        },
+        {
+            "title": "Expressive",
+            "content": "Julia's familiar mathematical syntax and metaprogramming features lead to highly expressive code",
```

Comment:
I think it's multiple dispatch, generator expressions, and the generous functionality of `Base` that makes Julia so expressive for me.

Reply:
Maybe we could dump "General" and talk about two different types of expressivity instead?

Reply:
Sounds good, what would you call them?

Maybe "Expressive" for what you described and "<something> syntax" for the other type?

Reply:
Flexible or natural syntax maybe?

Reply:
What would count as flexible syntax other than metaprogramming?

Reply:
I was thinking of the good unicode support for operators and identifiers. That plus operator overloading lets you write code that looks like math.

Reply:
```suggestion
            "content": "Multiple dispatch, generator expressions and metaprogramming features lead to highly expressive code",
```

Reply:
Sounds good to me.